packages/docusaurus-plugin-content-docs/src/cli.ts

```diff
@@ -15,24 +15,23 @@ import path from 'path';
 import type {PathOptions, SidebarOptions} from './types';
 import {transformSidebarItems} from './sidebars/utils';
 import type {SidebarItem, NormalizedSidebars, Sidebar} from './sidebars/types';
-import {loadUnprocessedSidebars, resolveSidebarPathOption} from './sidebars';
+import {loadSidebarsFile, resolveSidebarPathOption} from './sidebars';
 import {DEFAULT_PLUGIN_ID} from '@docusaurus/core/lib/constants';
 
 function createVersionedSidebarFile({
   siteDir,
   pluginId,
   sidebarPath,
   version,
-  options,
 }: {
   siteDir: string;
   pluginId: string;
   sidebarPath: string | false | undefined;
   version: string;
-  options: SidebarOptions;
 }) {
   // Load current sidebar and create a new versioned sidebars file (if needed).
-  const loadedSidebars = loadUnprocessedSidebars(sidebarPath, options);
+  // Note: we don't need the sidebars file to be normalized: it's ok to let plugin option changes to impact versioned sidebars
+  const loadedSidebars = loadSidebarsFile(sidebarPath);
```

Comment:
Tests are failing because category shorthands are significantly harder to parse and append version prefixes. Maybe it's a good time to drop version prefixes altogether?

Reply:
hmm that's annoying indeed

Going to restore the normalized sidebar in this PR for now but we should remove the versioning/normalization of the versioned sidebar copies sooner or later, legacy of the past

Reply:
hmmm actually can't find a good way to do this easily, so yes it seems we'll need this refactor right now 😅 will be simpler than not doing it